applications/registration-react/src/lib/registration-status.js

```diff
@@ -0,0 +1,4 @@
+const REGISTRATION_STATUS_PUBLISH = 'PUBLISH';
```

Comment:
er det ikke dataset modell? api og dataset har ikke jo samme set av status verdier.

Reply:
api har 'DRAFT' og 'PUBLISH'. Datasett får i tillegg 'APPROVE'. Bare datasett som får knapp for 'APPROVE'.

Reply:
jeg mente, at vi har her en ENUM, listen av lovlig valg, og den valg er forskjellig for dataset og api. Fra dette setning kan man si at man kan ikke ha en felles "registration-status.js".
I stedet må det være to domainmodules

dataset.js -> export const registrationStatus:{PUBLISH:'PUBLISH, ...};
api-registration.js -> export const registrationStatus:{PUBLISH:'PUBLISH, ...};